checklists/alz_checklist.en.json

```diff
@@ -240,6 +240,18 @@
             "training": "https://learn.microsoft.com/learn/modules/secure-aad-users-with-mfa/",
             "link": "https://learn.microsoft.com/azure/active-directory/authentication/concept-mfa-howitworks"
         },
+        {
+            "category": "Identity and Access Management",
+            "subcategory": "Identity",
```

Comment:
Should this be merged with or replace B03.14?

Reply:
I think it is two different ones: B3.14 is coming from a networking angle, where AAD Proxy can be used to replace VPN connectivity. This new one is for specific apps, where the goal is not necessarily providing access to remote users, but increasing the security of the app.

From that angle, it might fit better in the App Delivery checklist?

Reply:
I think so - it has less to do with the landing zone practices over-all, and more to do with the security of specific applications, which would fit in elsewhere in my opinion.